src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -93,41 +93,44 @@
+"ExposureDetection_LowRisk_Behavior_Subtitle" = "So verhalten Sie sich richtig:";
+
+"ExposureDetection_Guide_Hygiene" = "Bitte achten Sie jetzt besonders auf die üblichen Hygieneregeln.";
 
 "ExposureDetection_Guide_Vaccination" = "Lassen Sie sich gegen SARS-CoV-2 impfen, falls Sie noch nicht geimpft sind.";
 
 "ExposureDetection_Guide_Hands" = "Waschen Sie Ihre Hände regelmäßig mit Seife für 20 Sekunden.";
 
-"ExposureDetection_Guide_Mask" = "Tragen Sie einen Mund-Nasen-Schutz bei Begegnungen mit anderen Personen.";
+"ExposureDetection_Guide_Mask" = "Tragen Sie eine Maske bei Begegnungen mit anderen Personen.";
 
 "ExposureDetection_Guide_Distance" = "Halten Sie mindestens 1,5 Meter Abstand zu anderen Personen.";
 
 "ExposureDetection_Guide_Sneeze" = "Niesen oder husten Sie in die Armbeuge oder in ein Taschentuch.";
 
 "ExposureDetection_Guide_Ventilation" = "Lüften Sie Innenräume mehrmals täglich. Öffnen Sie dazu die Fenster einige Minuten lang so weit wie möglich (Stoßlüftung).";
 
-"ExposureDetection_Guide_Symptoms" = "Achten Sie auf COVID-19-typische Symptome. Symptome können in seltenen Fällen auch bei geimpften Personen auftreten. Sobald Symptome auftreten, lassen Sie sich testen.";
-
-"ExposureDetection_Guide_Home" = "Begeben Sie sich, wenn möglich, nach Hause bzw. bleiben Sie zu Hause.";
-
-"ExposureDetection_Guide_Hotline1" = "Bei Fragen zu Symptomen, Testmöglichkeiten und Quarantäne-Maßnahmen wenden Sie sich bitte an eine der folgenden Stellen:";
+"ExposureDetection_Guide_Symptoms" = "Achten Sie auf Symptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten. Falls Sie sich krank fühlen oder Symptome auftreten, wenden Sie sich bitte an Ihre Hausarztpraxis.";
 
-"ExposureDetection_Guide_Hotline2" = "Ihre Hausarztpraxis";
+"ExposureDetection_Guide_Home" = "Begeben Sie sich, wenn möglich, nach Hause bzw. bleiben Sie zu Hause und vermeiden Sie Begegnungen außerhalb Ihres Haushalts.";
 
-"ExposureDetection_Guide_Hotline3" = "den kassenärztlichen Bereitschaftsdienst unter der Telefonnummer 116 117";
+"ExposureDetection_Guide_Hotline" = "Bei sonstigen Fragen wenden Sie sich bitte an den kassenärztlichen Bereitschaftsdienst unter der Telefonnummer 116 117.";
 
-"ExposureDetection_Guide_Hotline4" = "Ihr Gesundheitsamt";
+"ExposureDetection_Guide_Vaccination_HighRisk" = "Sollten Sie bisher nicht geimpft sein, besprechen Sie nach der Testung die Impfung mit Ihrer Hausarztpraxis.";
 
-"ExposureDetection_Guide_Vaccination_HighRisk" = "Sollten Sie bisher nicht geimpft sein, lassen Sie sich nach Rücksprache mit Ihrer Hausarztpraxis oder dem kassenärztlichen Bereitschaftsdienst testen. Besprechen Sie ebenfalls die SARS-CoV-2-Impfung nach der Testung mit Ihrer Hausarztpraxis.";
+"ExposureDetection_Guide_Title" = "Lassen Sie sich testen, auch wenn Sie keine Symptome haben. Bei einer Warnung über ein erhöhtes Risiko (\"Rote Kachel\") haben Sie Anspruch auf einen kostenlosen Test. Dies muss nicht notwendigerweise ein PCR-Test sein.
```

Comment:
We are missing information for vaccinated users here. 
I suggest to add a new sentence "Die Handlungsempfehlungen für geimpfte Personen finden Sie [hier](https://www.coronawarn.app/de/faq/#vac_cert_red_warning)"

@UA-Team

Reply:
Hello @Ein-Tim, 
What exactly are you missing? The information is for vaccinated persons valid as well. E.g. "Achten Sie auf Symptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten."

Reply:
@KathrinLuecke But here it says:
"Lassen Sie sich testen, auch wenn Sie keine Symptome haben." 

Is this also the recommendation for vaccinated or recovered people? Because if yes, the FAQ entry https://www.coronawarn.app/en/faq/#vac_cert_red_warning is either out of date or wrong.

Reply:
@Ein-Tim I will clarify that. Probably it's a bit out of sync.

Reply:
@Ein-Tim The FAQ will be adapted as well once these new recommendations are live.
And indeed, all recommendations are valid for unvaccinated, as well as for vaccinated and recovered persons - right at the beginning it says "So verhalten Sie sich richtig, auch, wenn Sie geimpft oder genesen sind, und auch, wenn Sie keine Symptome haben:"

Reply:
@KathrinLuecke 

Okay, thanks for all your answers!